docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -156,258 +156,155 @@ Before proceeding, ensure that you have the following:
-   			name: **yugabyte/thirdparty-deps**
-      postgres:
-   			registry: "yugabyte/postgres"
-   			tag: 11.5
-   			name: **postgres**
-      prometheus:
-   			registry: "prom/prometheus"
-   			tag: **v2.2.1**
-   			name: **prom/prometheus**
-      nginx:
-   			registry: "yugabyte/nginx"
-   			tag: **1.17.4**
-   			name: nginx
-      dnsmasq:
-   			registry: "janeczku/go-dnsmasq/"
-   			tag: **release-1.0.7**
-   			name: **janeczku/go-dnsmasq
-   ```
-
-1. Install Helm chart or specify the container registry in YugabyteDB Anywhere cloud provider, as follows:
-
-   ```sh
-   helm install yugaware **.** -f values.yaml
-   ```
+- Login to [quay.io](https://quay.io/) to access the YugabyteDB private registry using the user name and password provided in the secret YAML file. To find the `auth` field, use `base64 -d` to decode the data inside the `yaml` file twice. In this field, the user name and password are separated by a colon. For example, `yugabyte+<user-name>:ZQ66Z9C1K6AHD5A9VU28B06Q7N0AXZAQSR`.
+
+  ```sh
+  docker login -u "your_yugabyte_username" -p "yugabyte_provided_password" quay.
+  docker search quay.io/yugabyte
+  ```
+
+- Install Helm to fetch the YugabyteDB Anywhere Helm chart on your machine. Follow to see [Helm Installation](https://helm.sh/docs/intro/install/).
+
+- Run the following `helm repo add` command to clone the [YugabyteDB charts repository](https://charts.yugabyte.com/).
+
+  ```sh
+  helm repo add yugabytedb https://charts.yugabyte.com
+  ```
+
+- Run the following `helm repo update` command to fetch the updates from the Chart repo.
+
+  ```sh
+  helm repo update
+  ```
+
+- Run the following command to search for the version.
+
+  ```sh
+  helm search repo yugabytedb/yugaware --version {{<yb-version version="preview" format="short">}}
+  ```
+
+  A similar output should appear:
+
+  ```output
+  NAME                            CHART VERSION   APP VERSION     DESCRIPTION
+  yugabytedb/yugaware             {{<yb-version version="preview" format="short">}}          {{<yb-version version="preview" format="build">}}    YugaWare is YugaByte Database's Orchestration a...
+  ```
+
+- Run the command to list all the available versions.
+
+  ```sh
+  helm search repo yugabytedb/yugaware -l
+  ```
+
+  A similar output should appear:
+
+  ```output
+  NAME                            CHART VERSION   APP VERSION     DESCRIPTION
+  yugabytedb/yugaware             2.17.0          2.17.0.0-b24    YugaWare is YugaByte Database's Orchestration a...
+  yugabytedb/yugaware             2.16.1          2.16.1.0-b50    YugabyteDB Anywhere provides deployment, orches...
+  yugabytedb/yugaware             2.16.0          2.16.0.1-b7     YugaWare is YugaByte Database's Orchestration a...
+  yugabytedb/yugaware             2.15.3          2.15.3.3-b1     YugaWare is YugaByte Database's Orchestration a...
+  yugabytedb/yugaware             2.15.2          2.15.2.1-b1     YugaWare is YugaByte Database's Orchestration a...
+  ```
```

Comment:
I think we don't need this, let's focus more on container images and less on the Helm commands/setup.